modules/elasticsearch/elasticsearch_test.go

```diff
@@ -108,21 +106,19 @@ func TestElasticsearch(t *testing.T) {
 
 			if tt.image != baseImage8 && err != nil {
 				if tt.passwordCustomiser != nil {
-					t.Fatal(err, "should access with authorised HTTP client.")
+					require.NoErrorf(t, err, "should access with authorised HTTP client.")
 				} else if tt.passwordCustomiser == nil {
-					t.Fatal(err, "should access with unauthorised HTTP client.")
+					require.NoErrorf(t, err, "should access with unauthorised HTTP client.")
 				}
 			}
 
 			if tt.image == baseImage8 {
-				if tt.passwordCustomiser != nil && err != nil {
-					t.Fatal(err, "should access with authorised HTTP client using TLS.")
+				if tt.passwordCustomiser != nil {
+					require.NoErrorf(t, err, "should access with authorised HTTP client using TLS.")
 				}
 				if tt.passwordCustomiser == nil && err == nil {
 					// Elasticsearch 8 should return 401 Unauthorized, not an error in the request
-					if resp.StatusCode != http.StatusUnauthorized {
-						t.Fatal("expected 401 status code for unauthorised HTTP client using TLS, but got", resp.StatusCode)
-					}
+					require.Equalf(t, http.StatusUnauthorized, resp.StatusCode, "expected 401 status code for unauthorised HTTP client using TLS, but got: %s", resp.StatusCode)
```

Comment:
chore: use stdlib instead of 401 status code

Reply:
Yes, given we added the new lint rule for that, I consider it could be used here. Wdyt about using it inside error messages in tests?